framework-plugins/lisk-framework-monitor-plugin/src/controllers/transactions.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Request, Response, NextFunction } from 'express';
+import { BaseChannel } from 'lisk-framework';
+import { SharedState } from '../types';
+
+interface PeerInfo {
```

Comment:
maybe move this interface to a `types` file since it is also be used in blocks endpoint

Reply:
moved and updated usages.